src/Sylius/Bundle/CustomerBundle/Resources/config/routing.yml

```diff
@@ -0,0 +1,6 @@
+sylius_register:
+    resource: @SyliusCustomerBundle/Resources/config/routing/registration.yml
+
+sylius_account:
+    resource: @SyliusCustomerBundle/Resources/config/routing/account.yml
+    prefix: /account
```

Comment:
Routing should be removed from `CustomerBundle` it is not used anywhere in new shop.